public/app/features/dashboard-scene/serialization/SaveDashboardDrawer.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import { SceneComponentProps, SceneObjectBase, SceneObjectState } from '@grafana/scenes';
+import { Drawer } from '@grafana/ui';
+import { SaveDashboardDiff } from 'app/features/dashboard/components/SaveDashboard/SaveDashboardDiff';
+import { jsonDiff } from 'app/features/dashboard/components/VersionHistory/utils';
+
+import { DashboardScene } from '../scene/DashboardScene';
+
+import { transformSceneToSaveModel } from './transformSceneToSaveModel';
+
+interface SaveDashboardDrawerState extends SceneObjectState {}
+
+export class SaveDashboardDrawer extends SceneObjectBase<SaveDashboardDrawerState> {
+  constructor(public dashboard: DashboardScene) {
+    super({});
+  }
+
+  onClose = () => {
+    this.dashboard.setState({ drawer: undefined });
+  };
+
+  static Component = ({ model }: SceneComponentProps<SaveDashboardDrawer>) => {
+    const originalVersion = transformSceneToSaveModel(model.dashboard.getOriginal());
+    const saveVersion = transformSceneToSaveModel(model.dashboard);
+
+    const diff = jsonDiff(originalVersion, saveVersion);
+
+    // let diffCount = 0;
+    // for (const d of Object.values(diff)) {
+    //   diffCount += d.length;
+    // }
```

Comment:
```suggestion
```

Reply:
yea, I know, we will need this at some point, for the tab change counter :)